extractor/src/json_encoder.py

```diff
@@ -13,5 +13,9 @@ class MyJSONEncoder(flask.json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, decimal.Decimal):
             # Convert decimal instances to strings
+            # If the decimal reprensents an integer, we want to avoid a trailing ".0"
+            nearest_int = obj.to_integral_exact()
```

Comment:
would be nice to have a small unit test for this ?

Reply:
actually there's better: `return str(obj.normalize())`

Reply:
Way better, indeed! Thanks 👍